### PR TITLE
New metrics

### PR DESCRIPTION
**Pull request by newhoggy**
* served.header.count
* served.block.count
* submissions.collected.count
* submissions.accepted.count
* submissions.rejected.count

### CODE REVIEW

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 1]
```diff
@@ -402,21 +404,27 @@ traceChainMetrics tr = Tracer $ \ev ->
...
-     traceI "blockNum"    blocks
-     traceI "slotInEpoch" slotInEpoch
-     traceI "epoch"       (unEpochNo epoch)
+     traceD tr meta "density"     (fromRational density)
+     traceI tr meta "slotNum"     slots
+     traceI tr meta "blockNum"    blocks
+     traceI tr meta "slotInEpoch" slotInEpoch
+     traceI tr meta "epoch"       (unEpochNo epoch)
+
+traceD :: Trace IO a -> LOMeta -> Text -> Double -> IO ()
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:416]: :+1:

**Review by Jimbo4350 — APPROVED** [review]
LGTM but I would get @deepfire to have a look as well.

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 2]
```diff
@@ -441,21 +454,52 @@ mkConsensusTracers
...
     , Consensus.chainSyncServerBlockTracer = tracerOnOff (traceChainSyncBlockServer trSel) verb "ChainSyncBlockServer" tr
     , Consensus.blockFetchDecisionTracer = tracerOnOff' (traceBlockFetchDecisions trSel) $
         annotateSeverity $ teeTraceBlockFetchDecision verb elidedFetchDecision tr
     , Consensus.blockFetchClientTracer = tracerOnOff (traceBlockFetchClient trSel) verb "BlockFetchClient" tr
-    , Consensus.blockFetchServerTracer = tracerOnOff (traceBlockFetchServer trSel) verb "BlockFetchServer" tr
+    , Consensus.blockFetchServerTracer = tracerOnOff' (traceBlockFetchServer trSel) $
+        Tracer $ \ev -> do
+          traceWith (annotateSeverity . toLogObject' verb $ appendName "BlockFetchServer" tr) ev
+          when (isTraceBlockFetchServerBlockCount ev) $ do
+            count <- STM.atomically $ STM.modifyTVar tBlocksServed (+1) >> STM.readTVar tBlocksServed
```
Comment (nfrisby) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:485]: Great! This is what I had in mind 👍

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**